modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/model/listener/test/DDMStructureModelListenerTest.java

```diff
@@ -300,6 +311,63 @@ private void _assertDDMFormFieldValuesMap(
 		}
 	}
 
+	private void _assertUpdateDataDefinitionStructureKeyChanged()
+		throws Exception {
+
+		JournalArticle journalArticle = _addJournalArticle();
+
```

Comment:
It is very strange for an `assert*` method to create and update the date it is asserting. Can you separate the part that updates stuff from the part that checks that the values are the expected ones?